taskfiles/k8s.yml

```diff
@@ -23,9 +23,19 @@ tasks:
           paths='./...' \
           crd \
           webhook \
-          rbac:roleName=manager-role \
-          output:crd:artifacts:config=config/crd/bases \
-          output:rbac:artifacts:config=config/rbac/bases/operator
+          output:crd:artifacts:config=config/crd/bases
+      - |
+        controller-gen \
+          object:headerFile="../licenses/boilerplate.go.txt" \
+          paths='./internal/controller/redpanda/...' \
+          rbac:roleName=redpanda-operator-role \
```

Comment:
Just curious if we specified the same roleName here, would kustomize merge the rules together for us? IIRC cloud is a consumer of our kustomize configs so this will have some degree of impact on them but only when they next bump their operator version.

Reply:
I don't think kustomize would merge rules together. Never the less I will verify.